packages/react-components/react-popover/library/src/components/PopoverSurface/renderPopoverSurface.tsx

```diff
@@ -11,10 +12,12 @@ export const renderPopoverSurface_unstable = (state: PopoverSurfaceState) => {
   assertSlots<PopoverSurfaceSlots>(state);
 
   const surface = (
-    <state.root>
-      {state.withArrow && <div ref={state.arrowRef} className={state.arrowClassName} />}
-      {state.root.children}
-    </state.root>
+    <Fade visible={state.open}>
```

Comment:
We should introduce a motion slot if we add an animation, otherwise there is no way to remove it/customize it.

Reply:
We're not adding any animation; we're just swapping the implementation of what was already there.

Reply:
We've seen migration to `react-motion` that doesn't add a motion slot:

https://github.com/microsoft/fluentui/pull/31516/files#diff-366d3b377f81066ef64955e6d27ba04b8fefc8eb642042bc44d737d222494c01R19